diff --git a/chrome/browser/safe_browsing/download_protection/check_client_download_request.h b/chrome/browser/safe_browsing/download_protection/check_client_download_request.h
index 1932ca453ecf8e5a5ef8870de839e008c6325f81..d529c36b180768fbf9aaa311ea7ea77eaebaee5c 100644
--- a/chrome/browser/safe_browsing/download_protection/check_client_download_request.h
+++ b/chrome/browser/safe_browsing/download_protection/check_client_download_request.h
@@ -48,6 +48,7 @@ class CheckClientDownloadRequest : public CheckClientDownloadRequestBase,
                                   ClientDownloadRequest::DownloadType* type);
 
  private:
+  friend class BraveCheckClientDownloadRequest;
   // CheckClientDownloadRequestBase overrides:
   bool IsSupportedDownload(DownloadCheckResultReason* reason,
                            ClientDownloadRequest::DownloadType* type) override;
